src/mesh/mechanical/solid/femMesh.cpp

```diff
@@ -163,7 +161,7 @@ void femMesh::check_boundary_conditions(Json::Value const& boundary_data) const
 {
     for (auto const& boundary : boundary_data)
     {
-        for (auto const& mandatory_field : {"Name", "Time", "Type", "Values"})
+        for (auto const& mandatory_field : {"Name", "Type"})
```

Comment:
Reminds me, I'll add in clang-format as a commit hook so we don't have these wasted commits on formatting.